build_extension.sh

```diff
@@ -6,7 +6,7 @@ mkdir -p build/Release
 CMAKE_CXX_COMPILER="g++"
 IFS='-' read -r -a TARGET_ARR_WRONG_ORDER <<< "$PLAT"
 SYS_V="${TARGET_ARR_WRONG_ORDER[1]}"
-TARGET_ARCH="${TARGET_ARR_WRONG_ORDER[2]}"
+TARGET_ARCH="x86_64"
```

Comment:
this won't allow to build for arm processors, right?
I'd rather check if the arch is 'unknown', and default to x86_64 (linux, windows) or universal2 (mac) if that's the case.